lighthouse-extension/gulpfile.js

```diff
@@ -14,7 +14,6 @@ const eslint = require('gulp-eslint');
 const livereload = require('gulp-livereload');
 const tap = require('gulp-tap');
 const zip = require('gulp-zip');
-const rename = require('gulp-rename');
 
 const audits = fs.readdirSync(path.join(__dirname, '../', 'lighthouse-core/audits/'))
```

Comment:
could we switch this to using `Runner.getAuditList()` (and `Runner.getGathererList` below)? That will keep these up to date as those change